udata/harvest/tests/test_dcat_backend.py

```diff
@@ -172,3 +173,63 @@ def test_hydra_legacy_paged_collection_pagination(self):
 
         job = source.get_last_job()
         self.assertEqual(len(job.items), 4)
+
+    @httpretty.activate
+    def test_failure_on_initialize(self):
+        url = DCAT_URL_PATTERN.format(path='', domain=TEST_DOMAIN)
+        httpretty.register_uri(httpretty.GET, url, body='should fail')
+        org = OrganizationFactory()
+        source = HarvestSourceFactory(backend='dcat',
+                                      url=url,
+                                      organization=org)
+
+        actions.run(source.slug)
+
```

Comment:
Like other tests, I try to respect Given/When/Then so I had a blank line between the initial state, the action and the different assertion blocks (there can be multiple assertions block so blank lines between assertions).
This is only for a better tests readability, to see the different logical steps

There is only one blank line before and one after, others before are not blank, this is the indentation